pkg/gardenlet/controller/shoot/seed_registration_control_test.go

```diff
@@ -298,7 +297,7 @@ var _ = Describe("SeedRegistrationReconciler", func() {
 				})
 
 				It("should create the correct ManagedSeed resource, all custom", func() {
-					shoot := newShoot("true,disable-dns,disable-capacity-reservation,protected,invisible,minimumVolumeSize=20Gi,apiServer.replicas=2,apiServer.autoscaler.minReplicas=2,apiServer.autoscaler.maxReplicas=5,blockCIDRs=169.254.169.254/32,shootDefaults.pods=100.96.0.0/11,shootDefaults.services=100.64.0.0/13,backup.provider=gcp,backup.region=europe-north1,use-serviceaccount-bootstrapping,with-secret-ref,featureGates.Logging=false,resources.capacity.shoots=100,loadBalancerServices.annotations.foo=bar,ingress.controller.kind=nginx")
+					shoot := newShoot("true,disable-dns,disable-capacity-reservation,protected,invisible,minimumVolumeSize=20Gi,apiServer.replicas=2,apiServer.autoscaler.minReplicas=2,apiServer.autoscaler.maxReplicas=5,blockCIDRs=169.254.169.254/32,shootDefaults.pods=100.96.0.0/11,shootDefaults.services=100.64.0.0/13,backup.provider=gcp,backup.region=europe-north1,use-serviceaccount-bootstrapping,with-secret-ref,resources.capacity.shoots=100,loadBalancerServices.annotations.foo=bar,ingress.controller.kind=nginx")
```

Comment:
Can you preserve this test/scenario but with just another feature gate?

Reply:
For this case, Since the feature gate was set to false, and by default logging is disabled, I guess this test scenario is preserved. PTAL?

Reply:
This test is not about the Logging feature gate value and its default. It is rather about the the parsing of feature gates in the `use-as-seed` annotation - Logging is just an example feature gate that is used for the test purpose to verify that a feature gate gets be parsed and gets set in the ManagedSeed spec.

If I go and introduce the following regression:
```diff
diff --git a/pkg/apis/core/v1beta1/helper/helper.go b/pkg/apis/core/v1beta1/helper/helper.go
index 4544ae83c..59f31c9e7 100644
--- a/pkg/apis/core/v1beta1/helper/helper.go
+++ b/pkg/apis/core/v1beta1/helper/helper.go
@@ -312,7 +312,6 @@ func parseShootedSeed(annotation string) (*ShootedSeed, error) {

        shootedSeed := ShootedSeed{
                LoadBalancerServicesAnnotations: parseShootedSeedLoadBalancerServicesAnnotations(settings),
-               FeatureGates:                    parseShootedSeedFeatureGates(settings),
        }

        apiServer, err := parseShootedSeedAPIServer(settings)
```

this test (with the change from this PR) will no longer fail (catch an issue).

Reply:
Thanks for the explanation. Updated it e7c61696dfb8caf9f93cd06f3bf4df92b10dcd09